plugins/mac-capture/mac-screen-capture.m

```diff
@@ -1009,6 +1050,14 @@ static bool content_settings_changed(void *data, obs_properties_t *props,
 		props, "show_hidden_windows",
 		obs_module_text("WindowUtils.ShowHidden"));
 
+	obs_properties_add_bool(props, "show_cursor",
+				obs_module_text("DisplayCapture.ShowCursor"));
+
+	obs_property_t *hide_obs = obs_properties_add_bool(
+		props, "hide_obs", obs_module_text("DisplayCapture.HideOBS"));
+	obs_property_set_modified_callback2(hide_obs, content_settings_changed,
+					    sc);
```

Comment:
I think this also needs to be in the `if (sc)` block below to avoid crashes when `sc` is null, e.g. by calling obs_get_source_properties on the source type.

Reply:
Yeah the modified callbacks shouldn't even be added if there is no valid data object.